modules/distribution-zones/build.gradle

```diff
@@ -37,6 +37,11 @@ dependencies {
     implementation libs.jetbrains.annotations
     implementation libs.fastutil.core
     implementation libs.auto.service.annotations
+    implementation(libs.jsonpath.core) {
+        //IDEA test runner doesn't apply Gradle dependency resolve strategy, this is just not implemented
```

Comment:
Not sure, that I understand how the first line connected with the second, could you explain?

Reply:
This is a copy-paste from the other module where JsonPath was introduced, I think it's important to exclude this libs here as well

Reply:
I see, thanks!